io/EptReader.cpp

```diff
@@ -180,33 +218,19 @@ void EptReader::handleOriginQuery()
         throwError("Failed lookup of origin: " + search);
     }
 
-    if (*m_queryOriginId >= files.size())
+    if (*m_queryOriginId >= sources.size())
     {
         throwError("Invalid origin ID");
     }
 
     // Now that we have our OriginId value, clamp the bounds to select only the
-    // data files that overlap the selected origin.
+    // data sources that overlap the selected origin.
 
-    const Json::Value& found(files[static_cast<Json::ArrayIndex>(
+    const Json::Value& found(sources[static_cast<Json::ArrayIndex>(
                 *m_queryOriginId)]);
 
     BOX3D q(toBox3d(found["bounds"]));
 
-    if (m_info->json().isMember("scale"))
-    {
-        // Make sure floating point precision doesn't make us discard points
-        // from a file query.  We'll be checking the OriginId anyway so we have
-        // leeway to bloat the query bounds as needed.
-        const auto& scale(m_info->scale());
-        q.minx -= std::fabs(scale[0]);
-        q.miny -= std::fabs(scale[1]);
-        q.minz -= std::fabs(scale[2]);
-        q.maxx += std::fabs(scale[0]);
-        q.maxy += std::fabs(scale[1]);
-        q.maxz += std::fabs(scale[2]);
-    }
-
     // Clip the bounds to the queried origin bounds.  Don't just overwrite it -
     // it's possible that both a bounds and an origin are specified.
     m_queryBounds.clip(q);
```

Comment:
- Assume arg `origin` selects a file with bounds `[0, 0, 20, 20]`.
- Assume arg `bounds` is `[0, 0, 10, 10]`.

Rather than resetting the query bounds to `[0, 0, 20, 20]`, instead we *clip* the existing query bounds to the intersection of the two bounds.  The thing I'm "not overwriting" is `m_queryBounds`, meaning I'm *not* doing `m_queryBounds = q`.